docs/start/about/index.mdx

```diff
@@ -189,6 +189,16 @@ HTTP response header names are case-insensitive; see the [HTTP specification on
 
 {/* | `X-Request-ID` |  |  |  | */}
 
+#### GraphQL-specific response headers
+
+| Header | Possible Values | Description | Example |
+|:-------|:----------------|:------------|:--------|
+| `X-Bc-Graphql-Complexity` | integer | Shows the complexity of the response content. If this exceeds `10000`, an error is returned. | `5241` |
```

Comment:
```suggestion
| `x-bc-graphql-complexity` | integer | Indicates the computed complexity score of the executed GraphQL query. If this exceeds `10000`, an error is returned. | `5241` |
``` 

The idea is that response has nothing to do with complexity, its calculated based on query (based on requested data)